interface/patient_file/summary/demographics.php

```diff
@@ -1013,7 +1013,7 @@ function setMyPatient() {
                 <div class="col-md-8">
                     <?php
 
-                    if ($deceased > 0) :
+                    if ($deceased['days_deceased'] !== null && $deceased['date_deceased'] !== "0") :
```

Comment:
hi @juggernautsei , how about 
```
if (!empty($deceased)) :

Reply:
@stephenwaite, I tried it, and no. It still shows deceased on the chart.